rust/src/fees.rs

```diff
@@ -0,0 +1,216 @@
+use super::*;
+// constants - are these right?
+const ABSTRACT_SIZE_VKEY: usize = 8;//64; // from 
+const ABSTRACT_SIZE_SIG: usize = 13;//64;
+const HL: usize = 4;//16; // really unsure about this one, found https://github.com/input-output-hk/cardano-base/blob/501d91e426ae84ce0ae056be38bd3db594af9fc2/cardano-crypto-class/src/Cardano/Crypto/Hash/MD5.hs#L16
```

Comment:
what's the result of this question?

Reply:
The result was that they used mock crypto for their test vectors for tx fee sizes. I removed all the tx size stuff from this PR anyway. We'll do it in another with the new way.